Assets/Scripts/UI/Canvas/PhotoBookSystem/Page.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System.Collections;
+using Framework.PhoneCamera;
+using Framework.ScriptableObjects;
 using TMPro;
 using UnityEngine;
 using UnityEngine.UI;
```

Comment:
```suggestion
﻿using System.Collections;
using TMPro;
using UnityEngine;
using UnityEngine.UI;

using Framework.PhoneCamera;
using Framework.ScriptableObjects;
```
[Code convention](https://github.com/Team-Swamp/Codename-BDLMTW/wiki/Code-convention#importing-namespaces)